resource/read.go

```diff
@@ -50,4 +66,14 @@ type ReadResponse struct {
 	// resource. An empty slice indicates a successful operation with no
 	// warnings or errors generated.
 	Diagnostics diag.Diagnostics
+
+	// Deferred indicates that Terraform should defer
+	// importing this resource.
```

Comment:
Read 😄 and similar suggestion:

```suggestion
	// Deferred indicates that Terraform should defer refreshing this
	// resource until a followup plan operation.
```